omero/users/index.txt

```diff
@@ -16,19 +16,19 @@ several Java client applications, as well as Python and C++ bindings
 and a Django-based web application.
 
 The OMERO clients are cross-platform. To run on your computer they require
-Java 1.7 or higher to be installed. This can easily be installed from
-http://java.com/en if it is not already included in your OS. The OMERO.insight
-client gets all of its information from a remote OMERO.server — the location
-of which is specified at login. Since this connection utilises a standard
-network connection, the client can be run anytime the user is connected to the
-internet.
+Java |javaversion| or higher to be installed. This can easily be installed
+from http://java.com/en if it is not already included in your OS. The
```

Comment:
`http://java.com/en` is a locale specific link, if you just use `http://java.com/` it should choose the language based on the browser's settings.